broadcaster/_backends/kafka.py

```diff
@@ -10,9 +11,11 @@
 
 
 class KafkaBackend(BroadcastBackend):
-    def __init__(self, url: str):
-        self._servers = [urlparse(url).netloc]
+    def __init__(self, urls: str | list[str]) -> None:
```

Comment:
Shouldn't this fail on Python 3.8 and 3.9? I was expecting this for backwards compatibility:

```suggestion
    def __init__(self, urls: typing.Union[str, typing.List[str]]) -> None:
```

Reply:
No, it uses future annotations.